browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -146,7 +149,8 @@ class BraveBrowserView::TabCyclingEventHandler : public ui::EventObserver,
 };
 
 BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
-    : BrowserView(std::move(browser)) {
+    : BrowserView(std::move(browser)),
+      accelerator_pref_manager_(browser_->profile()->GetPrefs()) {
```

Comment:
I think we should observe changes in AcceleratorPrefManager here, so that we can update accelerators for this browser when accelerators are changed from another window.

Reply:
Oh man, that is a good call. It completely slipped my mind that you could have multiple windows :facepalm:

Reply:
Mind taking another look? I've pushed the shortcut management into an `AcceleratorService`